app/src/main/java/org/p2p/wallet/swap/jupiter/repository/model/JupiterToken.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.swap.jupiter.repository.model
+
+import org.p2p.wallet.swap.jupiter.api.response.tokens.JupiterTokenExtensions
+
+data class JupiterToken(
```

Comment:
we can remove implementation detail - Jupiter and call it just SwapToken? 
just a nitpick